app/js/constants/situationsFamiliales.js

```diff
@@ -13,7 +13,7 @@ angular.module('ddsCommon').constant('situationsFamiliales', [
     },
     {
         value: 'relation_libre',
```

Comment:
No big side effect, but it induces change on the API. I the "Fixes #112" in the description still appropriate in this case ?